sdcm/nemesis.py

```diff
@@ -3348,9 +3348,9 @@ def decommission_post_action():
 
         ParallelObject(objects=[trigger, watcher], timeout=1200).call_objects()
         if new_node := decommission_post_action():
-            new_node.run_nodetool("rebuild", timeout=300, retry=3)
+            new_node.run_nodetool("rebuild")
```

Comment:
shouldn't we have a timeout anyway? like 30 min? or perhaps even 1 hour?

Reply:
I rather not blindly set a timeout, without having data.

I'm just reverting this into what is/was working before, we don't have timeout on any other place we did rebuild

@soyacz is working on a way we could define timeouts relative to the data-set size (or other constraints), and then we can circle back to such place and do proper work of collecting data and giving estimates.

setting 1 hour is same as setting 5min, I have on idea if which case it gonna break.

Reply:
As it is said by the @KnifeyMoloko there, it was part of the race fix:

```
This PR's commits address this and an additional potential source of racy behavior by: a) moving the log follower to be initiated before the trigger function is called and b) by introducing timeouts to the nodetool remoter calls.
```

@fruch are you sure that the above statement is false?

Reply:
the nodetool command under question was decommission, **not** the rebuild commands.
the rebuild commands are outside of the parallelObejct call

so yes I'm sure

Reply:
O.k.